src/components/common/tab/index.tsx

```diff
@@ -1,19 +1,47 @@
-import { CSSProperties, useState } from 'react';
+import { isAuthorizedState } from '@/stores/auth';
+import { isLoginModalVisibleState } from '@/stores/modal';
+import { selectedTabState } from '@/stores/tab';
+import { CSSProperties } from 'react';
+import { useNavigate } from 'react-router';
+import { useRecoilState } from 'recoil';
 import * as style from './style.css';
 
 export type TabProps = {
   items: string[];
+  originalItems: { [key: string]: string };
   type?: 'header' | 'modal';
   onClick: (item: string) => void;
   style?: CSSProperties;
 };
 
-const Tab = ({ items, type = 'header', onClick, ...props }: TabProps) => {
-  const [clickedItem, setClickedItem] = useState(items[0]);
+const Tab = ({
+  items,
+  originalItems,
+  type = 'header',
+  onClick,
+  ...props
+}: TabProps) => {
+  const [tabState, setTabState] = useRecoilState(selectedTabState);
+  const [isAuthorized, setIsAuthorized] = useRecoilState(isAuthorizedState);
```

Comment:
setIsAuthorized는 안쓰이는 것 같아 아래처럼 작성해주셔도 좋을 것 같아요!
```suggestion
  const isAuthorized = useRecoilValue(isAuthorizedState);
```